solace-message-router/azuredeploy.json

```diff
@@ -42,6 +42,7 @@
       },
       "allowedValues": [
         "East US",
+        "US Gov Virginia",
```

Comment:
I think you can remove this - the vm extension is not published to USGov... also update metadata.json to exclude USGov support (and remove the button from the readme (if present) - keep the badges)

Also remove the defaultValue as that tends to overload regions

Reply:
Thanks for advising, will remove.